README.md

```diff
@@ -152,3 +152,15 @@ at the command line, try:
 xfs_info /mount-point
 ```
 If you see `ftype=1` then you have d_type support.
+
+### Modifying cpu/memory/disk resources
+The default cpu/memory/disk resources when using virtual machines are provided
+by the [tripleo-quickstart-config/metalkube-nodes.yml](tripleo-quickstart-config/metalkube-nodes.yml) file:
+
+* 4 vCPU
+* 8 Gb RAM
+* 50 Gb HDD
+
+If required, the cpu/memory/disk can be customzied per host role
+(`openshift_master_memory`, `openshift_worker_memory`,...) by modifying that
+file prior to run the [04_setup_ironic.sh](04_setup_ironic.sh) script.
```

Comment:
Modifying the script isn't a great interface for this - if we want users to be able to change the vars, we should add a variable to override  tripleo-quickstart-config/metalkube-nodes.yml as I mentioned a couple of times before, or alternatively allow for a user vars file via a variable.

Perhaps I can fix that as a follow-up if it's not clear, but given the commit title it seems like we should expose vars to customize, or change the commit message and readme and just change the defaults?